scanner/image/db/konflux.Dockerfile

```
@@ -0,0 +1,48 @@
+FROM registry.redhat.io/rhel8/postgresql-15:latest
+
+LABEL \
+    com.redhat.component="rhacs-scanner-v4-db-container" \
+    com.redhat.license_terms="https://www.redhat.com/agreements" \
+    description="Scanner v4 Database Image for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.description="Scanner v4 Database Image for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.display-name="scanner-v4-db" \
+    io.openshift.tags="rhacs,scanner-v4-db,stackrox" \
+    maintainer="Red Hat, Inc." \
+    name="rhacs-scanner-v4-db-rhel8" \
+    source-location="https://github.com/stackrox/stackrox" \
+    summary="Scanner v4 DB for RHACS" \
+    url="https://catalog.redhat.com/software/container-stacks/detail/60eefc88ee05ae7c5b8f041c" \
+    # We must set version label to prevent inheriting value set in the base stage.
+    # TODO(ROX-20236): configure injection of dynamic version value when it becomes possible.
+    version="0.0.1-todo"
+
+USER root
+
+COPY --chown=postgres:postgres \
```

I realized just now that downstream Dockerfile does not change ownership of these files. If they were executable in build context, they should remain executable after copying. Let's not change the ownership here since it seems to work ok downstream.

```suggestion
COPY \
```

---

3db3526dfdce71fa9eec3086793c10bde0920bc3